baremetal/metal3machine_manager.go

```diff
@@ -1338,15 +1335,58 @@ func (m *MachineManager) SetNodeProviderID(ctx context.Context, bmhID, providerI
 		m.Log.Info("Target node is not found, requeuing")
 		return &RequeueAfterError{RequeueAfter: requeueAfter}
 	}
```

Comment:
indentation issue above here

Reply:
A bit unclear. can you elaborate ?

Reply:
if you expand the block just above, there is some indentation issues linked to the change apparently